core/src/mindustry/ui/dialogs/ResearchDialog.java

```diff
@@ -441,7 +483,7 @@ public void rebuildAll(){
                 button.resizeImage(32f);
                 button.getImage().setScaling(Scaling.fit);
                 button.visible(() -> node.visible);
-                if(!net.client()){
+                if(true){
```

Comment:
```suggestion 
button.clicked(() -> {
```
Its always true no need to have this if statement

Reply:
Dont blindly merge these suggestions a few dont work as they need further alteration (aka this one)